app/app-worker.ts

```diff
@@ -22,10 +22,13 @@ import {
 import { dispatchUrl } from './applications/duck';
 import ManifestProvider from './applications/manifest-provider/manifest-provider';
 import DistantFetcher from './applications/manifest-provider/distant-fetcher';
+// @ts-ignore no declaration file
 import { getForeFrontNavigationStateProperty } from './applications/utils';
+// @ts-ignore no declaration file
 import { setReleaseNotesSubdockVisibility } from './auto-update/duck';
 import * as bang from './bang/duck';
 import { AlertDialogProviderServiceImpl } from './dialogs/alertDialogProvider';
+// @ts-ignore no declaration file
```

Comment:
Why do we have so many ts-ignore ?

Reply:
> Why do we have so many ts-ignore ?

They all are *.JS files. IDE shows me the  warning like `Could not find a declaration file for module ...`